### DISCUSSION TITLE
Package as Android App 打包安卓应用程序

### DISCUSSION THREAD

**Discussion by XiangQinxi** [flet-dev/flet#527] [Ideas]
I don't want to use the Web to make flet applications, how to package Android App?
我不想使用Web来制作Flet应用程序，怎样打包成安卓应用程序

**Comment by ndonkoHenri** [comment]
This is not possible at the moment, but is in Flet's future plans. 

Have a look:
https://flet.dev/docs/guides/python/mobile-support
